lib/email_checker/mx.ex

```diff
@@ -1,4 +1,7 @@
 defmodule EmailChecker.MX do
+
+  defp max_timeout(), do: Application.get_env(:email_checker, :timeout_milliseconds, :infinity)
```

Comment:
In the style guide they suggest to not use parentheses when we don't have parameters. Any thought ?

https://github.com/niftyn8/elixir_style_guide/blob/master/README.md#syntax